holmes/core/tools.py

```diff
@@ -78,7 +78,7 @@ class Tool(ABC, BaseModel):
     )
     additional_instructions: Optional[str] = None
 
-    def get_openai_format(self):
+    def get_openai_format(self, model: Optional[str]):
```

Comment:
@Avi-Robusta maybe it will make sense to make a separate method like `get_gemini_format` and do check for `if "gemini"` somewhere else?